hr_timesheet_employee_allowed/models/account_analytic_line.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    @api.onchange('user_id')
+    def onchange_employee_id_line(self):
+        domain = {'employee_id': []}
+        hr_emp_obj = self.env['hr.employee']
+        employee_id = hr_emp_obj.search(
```

Comment:
For recordset objects, we shouldn't use `_ids` suffix. Use that only for variable with actual ID numbers.
Instead, use just `employee`.